monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -3957,6 +3958,48 @@ object Task extends TaskInstancesLevel1 {
   def parZip6[A1,A2,A3,A4,A5,A6](fa1: Task[A1], fa2: Task[A2], fa3: Task[A3], fa4: Task[A4], fa5: Task[A5], fa6: Task[A6]): Task[(A1,A2,A3,A4,A5,A6)] =
     parMap6(fa1,fa2,fa3,fa4,fa5,fa6)((a1,a2,a3,a4,a5,a6) => (a1,a2,a3,a4,a5,a6))
 
+  /**
+    * Generates `cats.FunctionK` values for converting from `Task` to
+    * supporting types (for which we have a [[TaskLift]] instance).
+    *
+    * See [[https://typelevel.org/cats/datatypes/functionk.html]].
+    */
+  def toK[F[_]](implicit F: TaskLift[F]): FunctionK[Task, F] =
+    new FunctionK[Task, F] {
```

Comment:
There's a macro method on FunctionK for this, IIRC `FunctionK.lift(F.taskLift)`
Also, kind-projector should let you write `λ[Task ~> F](F.taskLift)` too